### ISSUE TITLE
Tags and custom fields are missing in the articles in the smart search results

### ISSUE THREAD

**Issue by webfeuerflo** [joomla/joomla-cms#44404]
### Describe the solution you'd like

The Smart Search results for articles do not contain the corresponding tags and custom fields and these cannot be accessed.
It would be desirable if this information were also available in the results

**Comment by fgsw** [comment]
#### Tags Joomla 5:
![Image](https://github.com/user-attachments/assets/5d61c785-8398-4204-8219-d18456a3f6b5)

**Comment by Hackwar** [comment]
@fgsw That is an entry of type "tag" in the Smart Search index, but not the tags and custom fields belonging to an article as in your screenshot in the first result. This feature request is a valid request, even though I'm not sure if we should add this.

**Comment by webfeuerflo** [comment]
Exactly, the tags as separate results are there, but I would find it useful if they would be accessible/shown with the article result

**Comment by Hackwar** [comment]
I've been thinking about this and while I could live with adding tags to the taxonomies, we can't add tags and customfields to the result object. Besides the size argument for adding yet more data to the serialized object, my main objection is the problem of keeping the data in sync. If you need access to tags and custom fields, you can retrieve that data in a template override via the API.
